README.md

```diff
@@ -23,7 +23,8 @@ npm install serverless-slic-watch-plugin --save-dev
 
 
 The `topic` configuration must be configured with the ARN of an SNS Topic.
-Alarm configuration values may also be specified. These variables along with their defaults are shown in this sample configuration:
+Alarm configuration is _cascading_. This means any property is applied to lower levels where no override is present.
```

Comment:
```suggestion
Alarm configuration is _cascading_. This means that configuration properties are automatically propagated from parent to children nodes (unless an override is present at the given node).
```

Not sure if this might make the idea  a bit more clear. We could also add an example.